source/strings.tex

```diff
@@ -4760,8 +4760,8 @@
   constexpr basic_string_view() noexcept;
   constexpr basic_string_view(const basic_string_view&) noexcept = default;
   constexpr basic_string_view& operator=(const basic_string_view&) noexcept = default;
-  constexpr basic_string_view(const charT* str);
-  constexpr basic_string_view(const charT* str, size_type len);
+  constexpr basic_string_view(const value_type* str);
+  constexpr basic_string_view(const value_type* str, size_type len);
```

Comment:
Ok, then.   Let's not do that. :-).  What I really want is the typedefs at the top.  If we can agree that those are OK (and editorial), then I'll make a new PR with just those.